tox.ini

```diff
@@ -1,12 +0,0 @@
-[tox]
-envlist = py26, py27, py33, py34, py35, pypy
-
-[testenv:py26]
-deps = unittest2
-
-[testenv:py27]
-# deps = numpy
-[testenv]
-commands = {envpython} setup.py --quiet test
-deps = setuptools
-       six
```

Comment:
The issue is that circle CI was using this instead of the configuration given in circle.yml
I have to check that since I never used it before.